omniauth-office365.gemspec

```diff
@@ -15,8 +15,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = %w(lib)
   gem.version       = OmniAuth::Office365::VERSION
 
-  # Lock at 1.3.x due to https://github.com/intridea/omniauth-oauth2/issues/81
-  gem.add_dependency 'omniauth-oauth2', '~> 1.3.1'
+  gem.add_dependency 'omniauth-oauth2', '~> 1.6.0'
```

Comment:
We need this version of `omniauth-oauth2` to bump `omniauth-google-oauth2` to 0.6.0 which removes using the deprecated Google+ API endpoints

The issue that locking to `1.3.1` is fixed by overriding the callback_url